spaces/cover/index.json

```
@@ -12,7 +12,14 @@
         "symbol": "COVER",
         "decimals": 18
       }
-    }
+    },
+    {
+      "name": "uniswap",
+      "params": {
+        "address": "0x4688a8b1f292fdab17e9a90c8bc379dc1dbd8713",
+        "symbol": "COVER"
```

The symbol should be unique, you can do like this
```suggestion
        "symbol": "COVER (Uniswap)"
```

---

Committed, thank you!